### ISSUE TITLE
When Geocoder is on map, hidden text box gets focus first

### ISSUE THREAD

**Issue by regan-sarwas** [regan-sarwas/map-series-storytelling-template-js#3]
This appears as a missing focus ring in between visible map elements.  It is confusing, but not critical.  I think this has to do with the way ESRI is controlling/styling the geocoder control.  It works well otherwise.
